### ISSUE TITLE
Polyfill

### ISSUE THREAD

**Issue by nsehgal** [uber/h3-go#15]
How to convert a boundary (polygon) to hexagons with a resolution using uber h3 golang bindings? 

Similar to this in java~ 
```

 @Test
    public void testPolyfillLarge() {
        List<Long> hexagons = h3.polyfill(
                ImmutableList.<GeoCoord>of(
                        new GeoCoord(41.5, -70.4),
                        new GeoCoord(41.5, -69.8),
                        new GeoCoord(42.1, -69.8),
                        new GeoCoord(42.1, -70.4)
                ), null,
                10
        );

        assertTrue(hexagons.size() > 1000);
    }
```

Thanks

**Comment by isaacbrodsky** [comment]
Hi, I believe this is not implemented in the Go bindings yet, according to #2.

**Comment by jeffschoner** [comment]
This should be available now that #19 has been merged
